content/en/blog/2023/2023-03-01-introducing-kwok/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "Introducing KWOK: Kubernetes WithOut Kubelet"
+date: 2023-03-01
+slug: introducing-kwok
+---
+
+**Author:** Shiming Zhang (DaoCloud), Wei Huang (Apple), Yibo Zhuang (Apple)
+
+<img align="right" width="300em" src="/blog/2023/03/01/introducing-kwok/kwok.svg"/>
+
+Have you ever wondered how to set up a cluster of thousands of nodes just in seconds, how to simulate real nodes with a low resource footprint, and how to test your Kubernetes controller at scale without spending much on infrastructure?
+
+If you say "yes" to any of these questions, then you might be interested in KWOK, a toolkit that enables you to create a cluster of thousands of nodes in seconds.
+
+## What is KWOK?
+
+KWOK stands for Kubernetes WithOut Kubelet. So far, we provide two tools:
+
+- **kwok**: The core of this project. It simulates thousands of fake nodes.
+- **kwokctl**: A CLI to facilitate the creation and management of clusters simulated by **kwok**.
+
+## Why use KWOK?
+
+KWOK has several advantages:
+
+- **Speed**: You can create and delete clusters and nodes almost instantly, without waiting for boot or provisioning.
+- **Compatibility**: KWOK works with any tools or clients that are compliant with Kubernetes APIs, such as kubectl, helm, kui, etc.
+- **Portability**: KWOK has no special hardware or software requirements. You can run it on your laptop, desktop or any cloud instance that supports Docker. You can also run it alongside other Kubernetes clusters or tools.
```

Comment:
Is Docker required (could you also use Podman)? If there's no hard dependency on Docker, can we reword to make that clear?

Reply:
Updated, in addition to Docker, Nerdctl and binary are supported and Podman may be supported in the future.